application-diagram-ui/src/main/resources/Diagram/DiagramClass.xml

```diff
@@ -64,6 +64,28 @@
       <validationRegExp/>
       <classType>com.xpn.xwiki.objects.classes.BooleanClass</classType>
     </exportUsingSVG>
+    <pdfImageExportZoom>
+      <cache>0</cache>
+      <customDisplay/>
+      <disabled>0</disabled>
+      <displayType>select</displayType>
+      <hint>Specify the zoom applied to the PNG image used in PDF export. A bigger zoom will generate a bigger image with a better quality.</hint>
+      <multiSelect>0</multiSelect>
+      <name>pdfImageExportZoom</name>
+      <number>2</number>
+      <picker>0</picker>
+      <prettyName>pdfImageExportZoom</prettyName>
+      <relationalStorage>0</relationalStorage>
+      <separator> </separator>
+      <separators>|, </separators>
+      <size>1</size>
+      <sort>none</sort>
+      <unmodifiable>0</unmodifiable>
+      <validationMessage/>
+      <validationRegExp/>
+      <values>100%|200%|300%|400%</values>
```

Comment:
I think it's important to store the **raw data**. For me "400%" is the display value (could be "4x" also) while the raw value is ``4``. Also, having some default options to choose from is nice, but I would keep that for the (custom) displayer (which we don't need ATM IMO). So I would simply store the ``pdfImageExportZoom`` as a number (depending on what is supported by draw.io, i.e. if they are not limited to integer), allowing the user to put any value they want.

Reply:
But the interval shouldn't still be defined? Or just a suggestion is enough?
The user could introduce any number, and therefore the save process could take too much

Reply:
I think it's enough to mention in the property hint that the value should be > 0 and that the larger the value the larger the size of the generated image and the longer the save action. We could also mention that a value larger than 4 takes too much resources for very little extra value.

In any case I would let the user input any value, for the moment at least.